### PR TITLE
Fixes issue with iptables rule in DC/OS overlay

### PR DESCRIPTION
**Pull request by GoelDeepak**
## High-level description

1. Docker isolates its networks by adding an explicit DROP iptables rules for any cross communication. However, Overlay (which uses Docker network) allows seamless communication among Docker networks. Overlay by-passes Docker isolation by adding an explicit RETURN as the first iptables rule in the chain. However, due to a bug, Overlay missed adding that rule if Docker network is already
present. This patch fixes that issue

2. Bump cni plugin version to v0.7.1 (latest cni plugin release)

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-3697](https://jira.mesosphere.com/browse/DCOS_OSS-3697) Connectivity between bridged and overlay networks broken
 - [DCOS_OSS-3841](https://jira.mesosphere.com/browse/COPS-3474) Update CNI plugins to latest release

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): https://github.com/dcos/dcos-mesos-modules/compare/c8b12d7a32a718353417e687965d55a58d4e0c29...26d11daee118055ab0975b0eaa0a25b6cf1a35c3)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

File: packages/mesos-modules/buildinfo.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "single_source": {
     "kind": "git", 
     "git": "https://github.com/dcos/dcos-mesos-modules.git", 
-    "ref": "c8b12d7a32a718353417e687965d55a58d4e0c29", 
+    "ref": "26d11daee118055ab0975b0eaa0a25b6cf1a35c3",
```
Comment (urbanserj) [inline packages/mesos-modules/buildinfo.json]: https://github.com/dcos/dcos/pull/3155/commits/426d829d9e62eace94b50783ca88e63955479f96

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot label [Ready For Review]

**Pull request merged**